packages/connect-examples/expo-playground/app/components/common/ParameterInput.tsx

```
@@ -0,0 +1,551 @@
+      }
+      return String(val || '');
+    };
+
+    return (
+      <div key={field.name} className="flex items-center gap-2">
+        <div className="min-w-0 flex-shrink-0 w-32">{renderFieldLabel(field)}</div>
+        <div className="flex-1 min-w-0">
+          {field.type === 'textarea' ? (
+            <textarea
+              id={field.name}
+              value={getDisplayValue(value)}
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)